draft-ietf-trans-rfc6962-bis.md

```diff
@@ -2055,8 +2060,9 @@ action when a misissue is detected.
 A log can misbehave in several ways. Examples include: failing to incorporate a
 certificate with an SCT in the Merkle Tree within the MMD; presenting different,
 conflicting views of the Merkle Tree at different times and/or to different
-parties; and issuing STHs too frequently. Such misbehavior is detectable and
-[I-D.ietf-trans-threat-analysis] provides more details on how this can be done.
+parties; issuing STHs too frequently; and mutating the signature of a logged
+certificate. Such misbehavior is detectable and [I-D.ietf-trans-threat-analysis]
+provides more details on how this can be done.
 
```

Comment:
This should also mention "failure to furnish a chain containing the certifier of a logged certificate" as misbehavior.